modules/entity_version_history/entity_version_history.info.yml

```diff
@@ -1,7 +1,7 @@
 name: Entity version history
```

Comment:
```suggestion
name: 'Entity version history'
```

Reply:
Even without quotes it's a valid YAML syntax. Not to say that this is very, very out of scope.

Reply:
Drupal wraps module name and description by commas. We update it in other components during D9 compatibility fixes.

Reply:
Well, if Drupal does it, OK. Is there a Drupal coding standard in this regard?

Reply:
There isn't standard about it. But actually it wasn't decided how it should be in Drupal core. Currently it is inconsistent there also. Please have a look https://www.drupal.org/project/drupal/issues/2994928.